src/cmd/run.js

```diff
@@ -44,16 +47,18 @@ export type WatcherCreatorParams = {|
   desktopNotifications?: typeof defaultDesktopNotifications,
   ignoreFiles?: Array<string>,
   createFileFilter?: FileFilterCreatorFn,
+  addonReload?: typeof defaultAddonReload,
 |};
 
+
 export type WatcherCreatorFn = (params: WatcherCreatorParams) => Watchpack;
 
 export function defaultWatcherCreator(
   {
     addonId, client, sourceDir, artifactsDir, ignoreFiles,
```

Comment:
@rpl what's the best way to handle this in your opinion? Have a second set of types, so the function of both is clear like in `defaultWatcherCreator ({...WatcherCreatorParams}, {...ReloadParams})` or pass them like `defaultWatcherCreator (addend, client, {...ReloadParams})` ?